### ISSUE TITLE
Add option/ config value to disable the Alerts check

### ISSUE THREAD

**Issue by JimMackin** [salesagility/SuiteCRM#546]
It would be nice to be able to disable the Alerts check (even via setting a config value).

See https://suitecrm.com/forum/developer-help/6592-how-to-turn-off-disable-alerts-module

**Comment by samus-aran** [comment]
![](https://github.trello.services/images/mini-trello-icon.png) [Add option/ config value to disable the Alerts check](https://trello.com/c/XyVCsa5j/144-add-option-config-value-to-disable-the-alerts-check)

**Comment by samus-aran** [comment]
We have now set up a new home for suggestions at Trello.  All github issues that were labeled 'suggestion' have been moved and will be closed.  Certain ones will be progressed within the new Suggestion Box and may be re-opened.

Announcement of moving Suggestions:
https://suitecrm.com/forum/suggestion-box/13691-moving-suggestions-from-github-to

New SuiteCRM Suggestion Box
https://trello.com/b/Ht7LbMqw/suitecrm-suggestion-box
